### PR TITLE
RC_Channel:fix code formatting

### PR DESCRIPTION
**Pull request by Hwurzburg**
as I was creating #22924 I noticed how badly formatted the switch statements were so ran it thru astyle...NFC

### CODE REVIEW

File: libraries/RC_Channel/RC_Channel.cpp [hunk 1]
```diff
@@ -418,12 +418,19 @@ bool RC_Channel::read_6pos_switch(int8_t& position)
         return false;  // This is an error condition
     }
 
-    if      (pulsewidth < 1231) position = 0;
-    else if (pulsewidth < 1361) position = 1;
-    else if (pulsewidth < 1491) position = 2;
-    else if (pulsewidth < 1621) position = 3;
-    else if (pulsewidth < 1750) position = 4;
-    else position = 5;
+    if      (pulsewidth < 1231) {
```
Comment (khancyr) [inline libraries/RC_Channel/RC_Channel.cpp:421]: nitpick: too much space here

**Review by khancyr — APPROVED** [review]
LGTM, much better than before. NFC

File: libraries/RC_Channel/RC_Channel.cpp (hunk 1 above)
Comment (Hwurzburg) [reply libraries/RC_Channel/RC_Channel.cpp]: thanks, if astyle did not catch it, it wasn't caught...fixed

**Review by khancyr — APPROVED** [review]
LGTM

**Pull request merged**